project/StringArray detobject
---
string tarobject

